resin/models/device.py

```diff
@@ -692,34 +695,46 @@ def disable_device_url(self, uuid):
             endpoint=self.settings.get('pine_endpoint')
         )
 
-    def set_to_build(self, uuid, build):
+    def set_to_build(self, uuid, build_commit_hash):
         """
-        Set a device to specific build id.
+        Set a device to specific build commit hash.
 
         Args:
             uuid (str): device uuid.
-            build (str): build id.
+            build_commit_hash (str): build commit hash.
 
         Raises:
             DeviceNotFound: if device couldn't be found.
             ApplicationNotFound: if application couldn't be found.
+            FailedBuild: if build commit hash points to an error build.
```

Comment:
For similar reasons, this should be `if all builds for the commit hash failed.`

Reply:
Updated!